ScalaScript.sublime-syntax

```diff
@@ -51,9 +62,9 @@ contexts:
       scope: invalid.illegal.stray-bracket-end
 
     # Blocks
-    - match: '\{|(?i)\bevent\b'
+    - match: '\{|(?i)\bEvent\b'
       push: block
-    - match: '\}|(?i)\bend\b'
+    - match: '\}|(?i)\bEnd\b'
```

Comment:
I have never seen a block starting with Event and/or ending with End - is this still being used or is this deprecated?